libkbfs/mdserver_remote.go

```diff
@@ -79,25 +87,31 @@ var _ rpc.ConnectionHandler = (*MDServerRemote)(nil)
 func NewMDServerRemote(config Config, srvAddr string,
 	rpcLogFactory *libkb.RPCLogFactory) *MDServerRemote {
 	mdServer := &MDServerRemote{
-		config:     config,
-		observers:  make(map[tlf.ID]chan<- error),
-		log:        config.MakeLogger(""),
-		mdSrvAddr:  srvAddr,
-		rekeyTimer: time.NewTimer(MdServerBackgroundRekeyPeriod),
+		config:        config,
+		observers:     make(map[tlf.ID]chan<- error),
+		log:           config.MakeLogger(""),
+		mdSrvAddr:     srvAddr,
+		rpcLogFactory: rpcLogFactory,
+		rekeyTimer:    time.NewTimer(MdServerBackgroundRekeyPeriod),
+	}
+
+	mdServer.pinger = pinger{
+		name:    "MDServerRemote",
+		doPing:  mdServer.pingOnce,
```

Comment:
I'm not sure I love this design (bidirectional dependency), but I don't immediately see a cleaner way to do it.